src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1444,6 +1525,29 @@ else if("right".equalsIgnoreCase(element))
 			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("auto"),ElementInfo.VISUALMEDIA,Arrays.asList("le","pe")));
 			allelementVerifiers.remove(element);
 		}
+		else if("scroll-behavior".equalsIgnoreCase(element))
+		{
+			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("auto","smooth"),ElementInfo.VISUALMEDIA));
+			allelementVerifiers.remove(element);
+		}
+		else if("scroll-snap-align".equalsIgnoreCase(element))
+		{
+			auxilaryVerifiers[82]=new CSSPropertyVerifier(Arrays.asList("none","start","end","center"),null,null,null,true);
```

Comment:
Is there any reason for not describing this `auxilaryVerifiers` entry and the ones for `scroll-snap-type` in the static initializer together with the existing ones?

Reply:
There are lots of such cases, I think it is because if there's no use of this property in the CSS, then it does not need to initialize this entry.